sechub-integrationtest/src/test/java/com/mercedesbenz/sechub/integrationtest/scenario12/PDSWebScanJobScenario12IntTest.java

```diff
@@ -124,10 +124,27 @@ public void pds_web_scan_can_be_executed_and_works() throws Exception {
             assignTemplateToProject(templateId, project)
         ;
 
+        /* prepare 2 */
+        String maxScanConfiguration = """
+                {
+                    "apiVersion": "1.0",
+                    "webScan": {
+                        "url": "%s",
+                        "maxScanDuration": {
```

Comment:
Just a note:
when use invalid values inside our integration tests, it could be nice to add some comments here, why we use such invalid values.

for example we could add a comment line inside the configuration json (is accepted/ignored by our object mapper).

```suggestion
                        "maxScanDuration": {
                           // this is just for testing that even with illegal values, the configuration is accepted without errors
```